src/main/java/emissary/core/IBaseDataObject.java

```diff
@@ -66,6 +67,18 @@ enum MergePolicy {
      */
     void setChannelFactory(final SeekableByteChannelFactory sbcf);
 
+    /**
+     * Returns a new InputStream to the data that this BaseDataObject contains.
+     * 
+     * NOTE 1: Mutating the data elements of this IBaseDataObject while reading from the InputStream will have indeterminate
+     * results.
+     * 
+     * NOTE 2: The calling code is responsible for closing the returned InputStream.
+     * 
+     * @return a new InputStream to the data that this BaseDataObject contains.
+     */
+    InputStream newInputStream();
```

Comment:
Looking at how we're using this in the unit tests, I'm not loving this method name.  The name `newInputStream` works well on the `Channels` class because that method takes a parameter, but that's not the case here.

I think something like `getDataStream()` does a better job of conveying that this method provides streaming access to the BDO's data.